fastlane/Fastfile

```diff
@@ -381,4 +386,10 @@ SUPPORTED_LOCALES = [
     }
   end
 
+  private_lane :send_strings_for_translation do | options |
+    sh("cd .. && mkdir -p #{update_strings_path} && cp #{main_strings_path} #{update_strings_path} && git add #{update_strings_path}strings.xml")
+    sh("git diff-index --quiet HEAD || git commit -m \"Send strings to translation.\"")
```

Comment:
I run this with a dirty repo and it failed on me in a noisy way.

Have you considered using `ensure_git_status_clean`?

Reply:
Yep! This is a private lane, so it's not intended to be called on its own. And the git status is already verified in the caller lane: https://github.com/wordpress-mobile/release-toolkit/blob/4f40a68d3b56c48f13fd0af1c768a1a70d621826/lib/fastlane/plugin/wpmreleasetoolkit/actions/android/android_codefreeze_prechecks.rb#L36
so I don't think we need to check it again here.

Reply:
> This is a private lane, so it's not intended to be called on its own. And the git status is already verified in the caller lane

I didn't consider that. Makes a sense 👍